libraries/AP_Scripting/applets/Aerobatics/FixedWing/RateBased/README.md

```diff
@@ -0,0 +1,144 @@
+# Sport Aerobatics
+
+The lua script "sport_aerobatics.lua" implements scripted aerobatics,
+allowing fixed wing aircraft to execute a number of aerobatic
+manoeuvres either in AUTO mission or by triggering using pilot commands
+using RC switches.
+
+As always, but particularly with scripted aerobatics, test in SITL until 
+you understand the function and behaviour of each manouver. You will need 
+an appropriate aircraft, and be ready to take manual control if necessary!
+
+These tricks are rate-based vs the precision trajectory-based manouvers of the script in the TrajectoryBased directory above this one. However, many planes have difficulty completing those tricks unless they can sustain extended vertical climbs and knife-edges, especially in wind. These tricks do not try to maintain a geospatial track, but rather, just attitude rates. Even planes that cannot hold the 90 deg knife edge trick, will probably hold the 180 deg (inverted) one and do loops and rolls, and these tricks can be done in even strong wind, although will not be well shaped.
+
+## Available Manoeuvres
+
+The following table gives the available manoeuvres. Each manoeuvre has
+an ID number which is used in the AUTO mission or in the TRIKn_ID
+parameters (described below). The present ground track is used as the track for the trick.
+The "Turnaround" column indicates if the manoeuvre results in a course reversal, which impacts how it is used in 
+AUTO missions. Once the trick is completed, the mode that was being used at the start of the trick is restored. If the mode is CRUISE, its
+track and altitude are reset to the values present when the mode is restored. Tricks in AUTO missions require that they be performed between two waypoints to establish
+the ground track.
+
+| ID | Name                     | Arg1            | Arg2                       | Turnaround |
+| -- | ------------------------ | ------------_-- | -------------------------- | -----------|
+| 1  | Roll(s)                  | rollrate(dps)   | num rolls                  | No         |
+| 2  | Loop(s)/TurnAround       | pitchrate(dps)  | num loops or turnaround(0) | if num=0   |
```

Comment:
So I think pitch rate for this loop is different to bank angle for the scripted loop. I'm sure people would find it helpful if you could explain how this works. Ditto roll rate and yaw rate.

Reply:
rolling circle could use more info..I agree...since it uses two rates and they should be coordinated....not sure the pitch rate on a loop or roll rate on a roll needs much explanation though